dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -770,6 +803,7 @@ void CommandBarFlyoutCommandBar::OnKeyDown(
         bool isRightToLeft = m_primaryItemsRoot && m_primaryItemsRoot.get().FlowDirection() == winrt::FlowDirection::RightToLeft;
         bool isLeft = (args.Key() == winrt::VirtualKey::Left && !isRightToLeft) || (args.Key() == winrt::VirtualKey::Right && isRightToLeft);
         bool isRight = (args.Key() == winrt::VirtualKey::Right && !isRightToLeft) || (args.Key() == winrt::VirtualKey::Left && isRightToLeft);
+        bool isDown = (args.Key() == winrt::VirtualKey::Down && !isRightToLeft) || (args.Key() == winrt::VirtualKey::Up && isRightToLeft);
         bool isUp = (args.Key() == winrt::VirtualKey::Up && !isRightToLeft) || (args.Key() == winrt::VirtualKey::Down && isRightToLeft);
```

Comment:
So, right-to-left also flips up and down directions?

Reply:
@adrientetar, you were right to bring this up. We had a discussion here and ended up changing this so that Down and Up keys do 'logical' moves as opposed to 'physical' moves, so they become independent of the right-to-left setting. Thanks.